From: Maximiliano Curia <maxy@gnuservers.com.ar>
Subject: Set the icontheme for the unit_tests

--- a/autotests/signaturetest.cpp
+++ b/autotests/signaturetest.cpp
@@ -182,6 +182,7 @@
 void SignatureTester::testImages()
 {
     KPIMTextEdit::RichTextComposer edit;
+    QIcon::setThemeName(QStringLiteral("breeze"));
     edit.createActions(new KActionCollection(this));
     QString image1Path = KIconLoader::global()->iconPath(QStringLiteral("folder-new"), KIconLoader::Small, false);
     QImage image1, image2;
