From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Tue, 12 Jul 2016 20:11:10 +0200
Subject: Set the icontheme for the unit_tests

---
 autotests/signaturetest.cpp | 1 +
 1 file changed, 1 insertion(+)

Index: kidentitymanagement/autotests/signaturetest.cpp
===================================================================
--- kidentitymanagement.orig/autotests/signaturetest.cpp
+++ kidentitymanagement/autotests/signaturetest.cpp
@@ -184,6 +184,7 @@ void SignatureTester::testTextEditInsert
 void SignatureTester::testBug167961()
 {
     KPIMTextEdit::RichTextComposer edit;
+    QIcon::setThemeName(QStringLiteral("breeze"));
     edit.createActions(new KActionCollection(this));
     Signature sig;
     sig.setEnabledSignature(true);
